build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.flywaydb:flyway-core'
     implementation 'org.flywaydb:flyway-mysql'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation', version: '3.3.3'
```

Comment:
Perhaps you shouldn't indicate vesrion and use same version for this component that already defined for org.springframework.boot (3.3.4) group

Reply:
writes me that ``Label 'version' is required``

Reply:
implementation 'org.springframework.boot:spring-boot-starter-validation'